src/templates/admView.html

```diff
@@ -49,7 +49,29 @@
                         {% endif %}
                     </td>
                     <td>
-                        {{ attr.part_of_group.grouping_key or 'Not grouped' }}
+                        <form method="POST"
+                            action="{{ url_for('views.setGroup', attribute_id=attr.id ) }}">
+                            <select name="select_group">
+                                {% if attr.part_of_group.grouping_key %}    
+                                    {% for x in groups %}                               
+                                        {% if x.grouping_key == attr.part_of_group.grouping_key %}
+                                            <option value="{{ attr.part_of_group.grouping_key }}" selected hidden> 
+                                                {{ attr.part_of_group.grouping_key }} - {{ x.group_name }}
+                                            </option> 
+                                        {% endif %}
+                                    {% endfor %}
+                                    <option value="None">Not grouped</option> 
+                                {% else %}
+                                    <option value="None" selected hidden>Not grouped</option> 
```

Comment:
```suggestion
                                    <option value="None" selected>Not grouped</option> 
```
When the "Not grouped" option is hidden, a small bug appears where if you don't have a group selected, and you then select a group, there is no way to select the "not grouped" again without refreshing the page

Reply:
Thanks for noticing, now it's fixed!